app/views/signs/show.html.haml

```diff
@@ -16,9 +16,10 @@
       .glosses-container.glosses
         %h2.main_gloss= @sign.gloss_main
         %h2.main_gloss.maori-gloss= @sign.gloss_maori
-      = form_tag vocab_sheet_items_path do
-        = hidden_field_tag 'sign_id', @sign.id
-        = add_vocab_button('vocab_sheet.item.add', class: 'add_to_sheet')
+      = link_to "#", { class: "card__button add-to-vocab-btn ""add-to-vocab-sheet"" card__button--peach", "data-sign-id": @sign.id } do
```

Comment:
You've got some unnecessary double quotes around add-to-vocab-sheet on this line.
I think this will technically still function but both of the pairs before and after that class should be removed for readability.